Dockerfile

```diff
@@ -137,7 +138,7 @@ RUN \
     && make install \
     ; fi \
 # build janus-gateway
-    && git clone https://github.com/meetecho/janus-gateway.git ${BUILD_SRC}/janus-gateway \
+    && git clone --branch $JANUS_VERSION https://github.com/meetecho/janus-gateway.git ${BUILD_SRC}/janus-gateway \
```

Comment:
Thank you for the PR.

Although, looking at the tag `v0.2.4`, there has been [65 commits to master since this tag](https://github.com/meetecho/janus-gateway/compare/v0.2.4...master).

The way you have it in the `Dockerfile`, you will always hit this tag and not the latest commit.

I understand what you were trying to tie in. I will see what will be the best way to do so and include that in a future commit.